crates/krun/src/launch.rs

```diff
@@ -112,26 +147,37 @@ fn request_launch(
     command: PathBuf,
     command_args: Vec<String>,
     env: HashMap<String, String>,
-) -> Result<()> {
-    let mut stream = TcpStream::connect(format!("127.0.0.1:{server_port}"))?;
+) -> std::result::Result<(), LaunchError> {
```

Comment:
If `LaunchError` impls `Error` we could just use `anyhow::Result` here as usual...

Reply:
Done.